drivers/at30tse75x/at30tse75x.c

```diff
@@ -0,0 +1,312 @@
+        return -1;
+    }
+
+    return 0;
+}
+
+int at30tse75x_set_alarm_polarity(at30tse75x_t* dev, at30tse75x_alarm_polatity_t polarity)
+{
+    uint8_t config;
+    if(at30tse75x_get_config(dev, &config) != 0) {
+        return -1;
+    }
+
+    switch(polarity)
+    {
+    case AT30TSE75X_ALARM_ACTIVE_HIGH:
+        config |= AT30TSE75X_CONFIG__ALERT_POL_BIT;
+        break;
+    case AT30TSE75X_ALARM_ACTIVE_LOW:
+        config &= ~(AT30TSE75X_CONFIG__ALERT_POL_BIT);
+        break;
+    default:
+        return -1;
+    }
+
+    if(at30tse75x_set_config(dev, config) != 0) {
+            return -1;
+        }
+
+    return 0;
+}
+
+int at30tse75x_set_fault_tolerance(at30tse75x_t* dev, at30tse75x_fault_tolerance_t tolerance)
+{
+    if(tolerance < AT30TSE75X_ALARM_AFTER_1 ||
+       tolerance > AT30TSE75X_ALARM_AFTER_6) {
+        return -1;
+    }
+
+    uint8_t config;
+    if(at30tse75x_get_config(dev, &config) != 0) {
+        return -1;
+    }
+
+    config &= ~(AT30TSE75X_CONFIG__FTQ_MASK);
+    config |= tolerance << AT30TSE75X_CONFIG__FTQ_SHIFT;
+
+    if(at30tse75x_set_config(dev, config) != 0) {
+        return -1;
+    }
+
+    return 0;
+}
+
+int at30tse75x_set_limit_low(at30tse75x_t* dev, int8_t t_low)
+{
+    uint16_t tmp = (t_low << 8) | (0x00);
+    return at30tse75x_set_register(dev, AT30TSE75X_REG__LIMIT_LOW, &tmp);
+}
+
+int at30tse75x_set_limit_high(at30tse75x_t* dev, int8_t t_high)
+{
+    uint16_t tmp = (t_high << 8) | (0x00);
+    return at30tse75x_set_register(dev, AT30TSE75X_REG__LIMIT_HIGH, &tmp);
+}
+
+int at30tse75x_save_config(at30tse75x_t* dev)
+{
+    i2c_acquire(dev->i2c);
+    if(i2c_write_byte(dev->i2c, dev->addr, AT30TSE75X_CMD__SAVE_TO_NVRAM) != 1) {
+        i2c_release(dev->i2c);
+        return -1;
+    }
+    i2c_release(dev->i2c);
+    /* Wait for copy to complete */
+    xtimer_usleep(5000);
+    return 0;
+}
+
+int at30tse75x_restore_config(at30tse75x_t* dev)
+{
+    i2c_acquire(dev->i2c);
+    if(i2c_write_byte(dev->i2c, dev->addr, AT30TSE75X_CMD__RESTORE_FROM_NVRAM) != 1) {
+        i2c_release(dev->i2c);
+        return -1;
+    }
+    i2c_release(dev->i2c);
+    /* Wait for copy to complete */
+    xtimer_usleep(200);
+    return 0;
+}
+
+int at30tse75x_get_temperature(at30tse75x_t* dev, float* temperature)
+{
+    uint16_t tmp;
+    uint8_t config;
+    uint8_t resolution;
+
+    if(at30tse75x_get_config(dev, &config) != 0) {
+        return -1;
+    }
+
+    /* If sensor is shutdown trigger One-Shot mode*/
+    if(config & AT30TSE75X_CONFIG__SHUTDOWN_BIT) {
+
+        config |= AT30TSE75X_CONFIG__OS_BIT;
+        if(at30tse75x_set_config(dev, config) != 0) {
+            return -1;
+        }
+
+        /* Use resolution to calculate conversion time */
+        resolution = (config & AT30TSE75X_CONFIG__RESOLUTION_MASK) >>
+                        AT30TSE75X_CONFIG__RESOLUTION_SHIFT;
+
+        /* Wait until conversion is finished */
+        xtimer_usleep((uint32_t)(25000 << resolution));
```

Comment:
I understand that the conversion time is dependent from the resolution. But why shifting `25000`?